prime-dotnet-webapi/Services/Clients/Email/CHESClient.cs

```diff
@@ -0,0 +1,148 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.Mail;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+
+namespace Prime.Services
+{
+    public class CHESClient : BaseService, ICHESClient
```

Comment:
Since this is meant only to do web requests and doesn't need to talk to the database, you dont need BaseService or the DI in the constructor for it.